doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -792,99 +906,175 @@
    },
    "source": [
     "```python\n",
-    "corrected_energy = (e_total-e_kin)[::2]/N_PART - 2/3 * np.pi * DENSITY * (4* LJ_EPS * LJ_SIG**6) / LJ_CUT**3\n",
-    "sim_energy = corrected_energy.mean()\n",
-    "dsim_energy = corrected_energy.std()/np.sqrt(len(corrected_energy)-1)\n",
-    "print(\"The obtained potential energy per particle is {:1.3f} +/- {:1.3f}, which agrees well with\\\n",
-    "the value -5.38 given in [6]\".format(*np.round(np.array([sim_energy,dsim_energy]),3)))\n",
+    "rdf_obs = observables.RDF(ids1=system.part[:].id, min_r=R_MIN, max_r=R_MAX, n_r_bins=N_BINS)\n",
+    "rdf_acc = accumulators.MeanVarianceCalculator(obs=rdf_obs, delta_N=steps_per_uncorrelated_sample)\n",
+    "system.auto_update_accumulators.add(rdf_acc)\n",
     "```"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we don't need an elaborate integration loop, instead the RDFs are calculated and accumulated automatically"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system.integrator.run(N_SAMPLES*steps_per_uncorrelated_sample)"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
-    "solution2": "hidden"
+    "solution2": "hidden",
+    "solution2_first": true
    },
    "source": [
-    "We now plot the experimental radial distribution.\n",
-    "Empirical radial distribution functions have been determined for pure fluids <a href='#[7]'>[7]</a>, mixtures <a href='#[8]'>[8]</a> and confined fluids <a href='#[9]'>[9]</a>. We will compare our distribution $g(r)$ to the theoretical distribution $g(r^*, \\rho^*, T^*)$ of a pure fluid <a href='#[7]'>[7]</a>."
+    "**Exercise**\n",
+    "* Get the mean RDF (define ``rdf``) from the accmulator\n",
+    "* Get the histogram bin centers (define ``rs``) from the observable"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "code_folding": [],
     "solution2": "hidden"
    },
    "source": [
     "```python\n",
-    "fig3 = plt.figure(num=None, figsize=(10, 6),\n",
-    "                  dpi=80, facecolor='w', edgecolor='k')\n",
-    "fig3.set_tight_layout(False)\n",
-    "plt.plot(r, rdf, '-', color=\"#A60628\", linewidth=3, alpha=1, label='simulated')\n",
-    "\n",
-    "# Comparison to literature\n",
-    "# ========================\n",
-    "\n",
-    "# reduced units\n",
-    "T_star = TEMPERATURE / LJ_EPS\n",
-    "rho_star = DENSITY * LJ_SIG**3\n",
-    "\n",
-    "# expression of the factors Pi from Equations 2-8 with coefficients qi from Table 1\n",
-    "# expression for a,g\n",
-    "P = lambda q1, q2, q3, q4, q5, q6, q7, q8, q9: \\\n",
+    "rdf = rdf_acc.get_mean()\n",
+    "rs = rdf_obs.bin_centers()\n",
```

Comment:
isn't this name a bit short? Also i had to look for the plot label to understand the name. Personally i think `bin_centers` would be more expressive but the development from `r` to `rs` is already a step in the right direction.